docs/features.md

```diff
@@ -156,53 +127,45 @@ Default settings:
 
 Default settings:
 
-```json
-"GZIP": {
-  "isEnabled": true,
-}
+```yaml
+GZIP:
+  isEnabled: true
 ```
 
 - **HIDDEN_NAMESPACES** – is used to define a list of Namespaces that are considered system, and are hidden by default.
 
 Default settings:
 
-```bash
-"HIDDEN_NAMESPACES": {
-  "isEnabled": true,
-  "config": {
-    "namespaces": [
-      "compass-system",
-      "istio-system",
-      "knative-eventing",
-      "knative-serving",
-      "kube-system",
-      "kyma-backup",
-      "kyma-installer",
-      "kyma-integration",
-      "kyma-system",
-      "natss",
-      "kube-node-lease",
-      "serverless-system"
-    ]
-  }
-}
+```yaml
+HIDDEN_NAMESPACES:
+  isEnabled: true
+  config:
+    namespaces:
+      - compass-system
+      - istio-system
+      - knative-eventing
+      - knative-serving
+      - kube-system
+      - kyma-backup
+      - kyma-installer
+      - kyma-integration
+      - kyma-system
+      - natss
+      - kube-node-lease
+      - serverless-system
```

Comment:
Do we need to use the "-" character in lines 145-155?
In other places with table elements, we use "-" only next to the first element.

Reply:
This is an array, so yes we need it :p